packages/fxa-auth-server/test/local/senders/mjml-emails.ts

```diff
@@ -1543,6 +1744,597 @@ describe('lib/senders/mjml-emails:', () => {
+          privacyUrl: 'privacyUrl',
+          subject: 'subject',
+          supportLinkAttributes: 'supportLinkAttributes',
+          supportUrl: 'supportUrl',
+          timestamp: 'timestamp',
+        },
+        uid: 'foo',
+      };
+      mailer.sesConfigurationSet = 'wibble';
+
+      return mailer.send(message).then((response) => {
+        assert(mailer.emailService.sendMail.calledOnce);
+        assert(!mailer.mailer.sendMail.called);
+
+        const args = mailer.emailService.sendMail.args[0];
+
+        assert.equal(args.length, 2);
+        assert.equal(args[0].to, 'emailservice.foo@restmail.net');
+        assert.equal(args[0].subject, 'subject');
+        assert.equal(args[0].provider, 'ses');
+
+        const headers = args[0].headers;
+
+        assert.equal(headers['X-Template-Name'], 'verifyLogin');
+        assert.equal(headers['X-Email-Service'], 'fxa-email-service');
+        assert.equal(headers['X-Email-Sender'], 'ses');
+        assert.equal(headers['X-Uid'], 'foo');
+
+        const expectedSesMessageTags = sesMessageTagsHeaderValue(
+          message.template,
+          'fxa-email-service'
+        );
+        assert.equal(headers['X-SES-MESSAGE-TAGS'], expectedSesMessageTags);
+        assert.equal(headers['X-SES-CONFIGURATION-SET'], 'wibble');
+
+        assert.equal(typeof args[1], 'function');
+      });
+    });
+
+    it('does not send request to fxa-email-service when the email pattern is not right', () => {
+      const message = {
+        email: 'foo@restmail.net',
+        subject: 'subject',
+        template: 'verifyLogin',
+        templateValues: {
+          action: 'action',
+          clientName: 'clientName',
+          device: 'device',
+          email: 'emailservice.foo@restmail.net',
+          ip: 'ip',
+          link: 'link',
+          location: 'location',
+          oneClickLink: 'oneClickLink',
+          passwordChangeLink: 'passwordChangeLink',
+          passwordChangeLinkAttributes: 'passwordChangeLinkAttributes',
+          privacyUrl: 'privacyUrl',
+          subject: 'subject',
+          supportLinkAttributes: 'supportLinkAttributes',
+          supportUrl: 'supportUrl',
+          timestamp: 'timestamp',
+        },
+        uid: 'foo',
+      };
+
+      return mailer.send(message).then((response) => {
+        assert(!mailer.emailService.sendMail.called);
+        assert(mailer.mailer.sendMail.calledOnce);
+        const args = mailer.mailer.sendMail.args[0];
+        assert.equal(args.length, 2);
+        assert.equal(args[0].to, 'foo@restmail.net');
+        assert.equal(args[0].subject, 'subject');
+        assert.equal(args[0].headers['X-Template-Name'], 'verifyLogin');
+        assert.equal(args[0].headers['X-Uid'], 'foo');
+        assert.equal(args[0].provider, undefined);
+        assert.equal(typeof mailer.mailer.sendMail.args[0][1], 'function');
+      });
+    });
+
+    it('sends request to fxa-email-service when selectEmailServices tells it to', () => {
```

Comment:
Hmm I didn't dig into this PR at all but we removed `fxa-email-service` in #10558. 🤔